backend/db/config.py

```diff
@@ -6,6 +6,8 @@
 
 def invert_list_dict(d1: Dict[str, List[str]]) -> Dict[str, List[str]]:
     """Invert a dictionary with lists as values"""
+    if any(not isinstance(value, list) for value in d1.values()):
+        raise ValueError('Dictionary values must be lists')
```

Comment:
FYI: catches invalid config type. Honestly I should support 'string' instead of 'list' if dependency is a single table though.